packages/nextjs/hooks/scaffold-eth/useScaffoldEventSubscriber.ts

```diff
@@ -1,7 +1,7 @@
 import { useTargetNetwork } from "./useTargetNetwork";
 import { Abi, ExtractAbiEventNames } from "abitype";
 import { Log } from "viem";
-import { useContractEvent } from "wagmi";
+import { useWatchContractEvent } from "wagmi";
 import { addIndexedArgsToEvent, useDeployedContractInfo } from "~~/hooks/scaffold-eth";
 import { ContractAbi, ContractName, UseScaffoldEventConfig } from "~~/utils/scaffold-eth/contract";
 
```

Comment:
We need to update this comment to v2 changes

Reply:
Ohhh tysm Pablo !! completely missed it ! Updated it at https://github.com/scaffold-eth/scaffold-eth-2/pull/700/commits/c33edd4504aa63ffbaa86c49705f5e8cbef20df3

Also renamed this hook inline with new wagmi naming at https://github.com/scaffold-eth/scaffold-eth-2/pull/700/commits/3a8143bac2368825e37cf1006ce4717655b79bfa